src/Microsoft.AspNet.Session/SessionMiddlewareExtensions.cs

```diff
@@ -1,45 +1,16 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using Microsoft.AspNet.Session;
-using Microsoft.Framework.Caching.Distributed;
-using Microsoft.Framework.Caching.Memory;
-using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Internal;
-using Microsoft.Framework.Logging;
-using Microsoft.Framework.OptionsModel;
 
 namespace Microsoft.AspNet.Builder
 {
     public static class SessionMiddlewareExtensions
     {
-        public static IApplicationBuilder UseInMemorySession([NotNull] this IApplicationBuilder app, IMemoryCache cache = null, Action<SessionOptions> configure = null)
+        public static IApplicationBuilder UseSession([NotNull] this IApplicationBuilder app)
         {
-            return app.UseDistributedSession(new LocalCache(cache ?? new MemoryCache(new MemoryCacheOptions())), configure);
```

Comment:
You also need to remove Session's direct dependency on the Caching.Memory package (and add it to tests instead).